pkg/skaffold/config/global_config.go

```diff
@@ -45,3 +46,9 @@ type SurveyConfig struct {
 	LastTaken     string `yaml:"last-taken,omitempty"`
 	LastPrompted  string `yaml:"last-prompted,omitempty"`
 }
+
+// UpdateConfig is the update config information
+type UpdateConfig struct {
+	// TODO (tejaldesai) Move update-check config within this struct
```

Comment:
what else needs to be done for this?

Reply:
That todo is to move the `update-check` boolean defined at the  top level into this struct.